aws_auth.tf

```diff
@@ -20,7 +20,16 @@ resource "null_resource" "update_config_map_aws_auth" {
 
 data "aws_caller_identity" "current" {}
 
-data "template_file" "worker_role_arns" {
```

Comment:
Can we leave this resource named the same? And call the second one `worker_launch_template_role_arns` to keep it consistent?